### PR TITLE
US-191 : archi v2

### PR DESCRIPTION
**Pull request by bdavidxyz**

### CODE REVIEW

File: api/tests/integration/controllers/assessments_test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    const options = { method: "POST", url: "/api/assessments", payload: {
+
+    }};
+
+    it("should return 201 HTTP status code", function (done) {
+      server.injectThen(options).then((response) => {
+        expect(response.statusCode).to.equal(201);
+        done();
+      });
```
Comment (mackwic) [inline api/tests/integration/controllers/assessments_test.js]: ça fait beaucoup de duplication tout ça...

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

File: api/tests/integration/controllers/courses_test.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const server = require('../../../server');
+const _ = require('lodash');
+const util = require('util');
```
Comment (mackwic) [inline api/tests/integration/controllers/courses_test.js:3]: du coup t'as oublié de le supprimer :P

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Comment by twickham** [comment]
I've deployed this PR to http://191-setup-archi-v2.pix.beta.gouv.fr. Please check it out

**Pull request merged**